src/patternfly/components/Button/button.scss

```diff
@@ -110,6 +110,7 @@
 
   // Styles for an icon in button
   --pf-c-button__icon--MarginRight: var(--pf-global--spacer--xs);
+  --pf-c-button__icon--MarginLeft: var(--pf-global--spacer--xs);
```

Comment:
Can we rename this to `--pf-c-button__text--icon--MarginLeft`? The existing var would be a conflict if we ever wanted to add a regular `margin-left` to `--pf-c-button__icon`.

Reply:
yep good point!